### PR TITLE
Fix kernels of notebooks, add notebook for use from colab

### PR DESCRIPTION
**Pull request by jimdowling**
Lots of notebooks now do not have the correct kernel, since when i updated them for examples.hopsworks.ai.

### CODE REVIEW

**Comment by moritzmeister** [comment]
Can you please make sure this is based on the latest version of master? It seems you are pulling notebooks back in that were deleted before.

**Review by robzor92 — CHANGES_REQUESTED** [review]
Can you also fix all the cells that you added for the Hugo tool to not be of type raw? Otherwise we will not be able to run the notebooks in jobs.

**Comment by jimdowling** [comment]
> Can you please make sure this is based on the latest version of master? It seems you are pulling notebooks back in that were deleted before.

Which ones? I can't see any new files apart from the colab notebook?

**Comment by jimdowling** [comment]
> Can you also fix all the cells that you added for the Hugo tool to not be of type raw? Otherwise we will not be able to run the notebooks in jobs.

They have to be of type raw to work with the python script that converts them to .md files and publishes them on the webpage.
The raw cell is not a problem for notebooks i have tried out. At least for pyspark notebooks. Can you give me an example of a notebook where raw conversion doesn't work, and i can look at it?

**Comment by jimdowling** [comment]
I fixed all the changes - changing the first cell to markdown worked ok, not raw. You can now run the notebooks again as jobs.

**Comment by jimdowling** [comment]
> Can you please make sure this is based on the latest version of master? It seems you are pulling notebooks back in that were deleted before.

It is based on master. There was an old s3 notebook that i found that i removed.

File: notebooks/featurestore/hsfs/time_travel/time_travel_python.ipynb [hunk 1]
```diff
@@ -1315,7 +1315,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python",
    "language": "python",
    "name": "python3"
```
Comment (moritzmeister) [inline notebooks/featurestore/hsfs/time_travel/time_travel_python.ipynb:1320]: This is a PySpark notebook, even though it's called `time_travel_python` it's just to distinguish it from the Scala version of the notebook.

File: notebooks/featurestore/hsfs/training/training-data-coalesced.ipynb [hunk 2]
```diff
@@ -67,7 +67,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python",
    "language": "python",
    "name": "python3"
```
Comment (moritzmeister) [inline notebooks/featurestore/hsfs/training/training-data-coalesced.ipynb:72]: I think this should also be a PySpark notebook.

File: notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_mnist_example.ipynb [hunk 3]
```diff
@@ -241,7 +241,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python",
    "language": "python",
    "name": "python3"
```
Comment (moritzmeister) [inline notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_mnist_example.ipynb:246]: distributed training needs PySpark kernel too, no?

**Review by moritzmeister — CHANGES_REQUESTED** [review]
I think these three notebooks should have the PySpark kernel, too.

File: notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_mnist_example.ipynb (hunk 3 above)
Comment (robzor92) [reply notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_mnist_example.ipynb:246]: It also needs it @jimdowling

Comment (jimdowling) [reply notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_mnist_example.ipynb:246]: Ok, will fix.

File: notebooks/featurestore/hsfs/time_travel/time_travel_python.ipynb (hunk 1 above)
Comment (jimdowling) [reply notebooks/featurestore/hsfs/time_travel/time_travel_python.ipynb]: Fixed

File: notebooks/featurestore/hsfs/training/training-data-coalesced.ipynb (hunk 2 above)
Comment (jimdowling) [reply notebooks/featurestore/hsfs/training/training-data-coalesced.ipynb]: Fixed

File: notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_mnist_example.ipynb (hunk 3 above)
Comment (jimdowling) [reply notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_mnist_example.ipynb]: Fixed.

**Comment by jimdowling** [comment]
Thanks for spotting the mistakes.

**Review by moritzmeister — APPROVED** [review]

**Comment by jimdowling** [comment]
> Can you also fix all the cells that you added for the Hugo tool to not be of type raw? Otherwise we will not be able to run the notebooks in jobs.

These are now all fixed.

File: notebooks/hive/PyHive.ipynb [hunk 4]
```diff
@@ -60,9 +64,34 @@
...
...hops.transaction.handler.RequestHandler.handle(RequestHandler.java:68)\\n\\tat org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkAccess(FSNamesystem.java:9017)\\n\\tat org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.checkAccess(NameNodeRpcServer.java:1386)\\n\\tat org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.checkAccess(ClientNamenodeProtocolServerSideTranslatorPB.java:1294)\\n\\tat org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)\\n\\tat org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:524)\\n\\tat org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1025)\\n\\tat org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:868)\\n\\tat org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:814)\\n\\tat java.security.AccessController.doPrivileged(Native Method)\\n\\tat javax.security.auth.Subject.doAs(Subject.java:422)\\n\\tat org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1821)\\n\\tat org.apache.hadoop.ipc.Server$Handler.run(Server.java:2900)\\n)'), operationHandle=None)\n",
```
Comment (robzor92) [inline notebooks/hive/PyHive.ipynb:74]: There is an stacktrace in this notebook?

File: notebooks/ml/Benchmarks/benchmark.ipynb [hunk 5]
```diff
@@ -192,7 +192,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python",
```
Comment (robzor92) [inline notebooks/ml/Benchmarks/benchmark.ipynb:195]: This should be PySpark too.

File: notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_simulated_data_example.ipynb [hunk 6]
```diff
@@ -209,7 +209,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python",
```
Comment (robzor92) [inline notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_simulated_data_example.ipynb:212]: This should be PySpark

File: notebooks/ml/Inference/Batch_Inference_Imagenet_Spark.ipynb [hunk 7]
```diff
@@ -37,9 +37,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 40,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```
Comment (robzor92) [inline notebooks/ml/Inference/Batch_Inference_Imagenet_Spark.ipynb:42]: Please remove stacktrace

**Review by robzor92 — CHANGES_REQUESTED** [review]

File: notebooks/ml/Benchmarks/benchmark.ipynb (hunk 5 above)
Comment (jimdowling) [reply notebooks/ml/Benchmarks/benchmark.ipynb:195]: done

File: notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_simulated_data_example.ipynb (hunk 6 above)
Comment (jimdowling) [reply notebooks/ml/Distributed_Training/mirrored_strategy/mirroredstrategy_simulated_data_example.ipynb]: Done

File: notebooks/ml/Benchmarks/benchmark.ipynb (hunk 5 above)
Comment (jimdowling) [reply notebooks/ml/Benchmarks/benchmark.ipynb]: Done

**Review by robzor92 — APPROVED** [review]

**Pull request merged**